cmd/core-command/Dockerfile

```diff
@@ -49,5 +51,13 @@ WORKDIR /
 COPY --from=builder /edgex-go/cmd/core-command/Attribution.txt /
 COPY --from=builder /edgex-go/cmd/core-command/core-command /
 COPY --from=builder /edgex-go/cmd/core-command/res/configuration.toml /res/configuration.toml
-ENTRYPOINT ["/core-command"]
-CMD ["-cp=consul.http://edgex-core-consul:8500", "--registry", "--confdir=/res"]
+
+ENV CONSUL_HOST=edgex-core-consul
+ENV CONSUL_PORT=8500
+
+# setup entrypoint script
+COPY --from=builder /edgex-go/cmd/core-command/entrypoint.sh /usr/local/bin/
```

Comment:
Just copy to `/docker-entrypoint.sh` like with the executable. This is more standard. Same for all the other Dockerfiles.
https://github.com/Kong/docker-kong/blob/b4c835f84d56a2d2d69c2780a1a409177ea0844f/alpine/Dockerfile#L43

Reply:
i am fine with that.